### ISSUE TITLE
Spacing Bug when Adjusting Artboard Sizd

### ISSUE THREAD

**Issue by nickm717** [DWilliames/paddy2-beta#28]
Attaching a video to be most clear, but when adjust the size of an artboard containing slightly complex padding/stacked symbols, the spacing is messed up.

Feel free to ask more question, or I can provide the file.

https://www.dropbox.com/s/mhqx247007s4ztg/Paddy2-ArtboardAdjustBug.mov?dl=0

**Comment by nickm717** [comment]
Sure! I can share that file with you privately. Just DM'd you a link on Twitter. 👍

**Comment by richardsison** [comment]
Yeah I get this issue too. When I resize an artboard, a bunch of my "paddy" groups will just have some resize randomly. It usually just stretches outwards to the bounds of the parent group. But sometimes it just nudges across (seemingly) randomly. Moving and resizing a containing group does nothing, but for some reason changing the artboard does.

**Comment by richardsison** [comment]
Here's an example of the bug for me: https://d.pr/i/QOHMbN

1. Resize artboard = triggers something in the paddy layers

The weird behaviour is what follows…

2. Undo
3. Resize Artboard again. No bugs this time.

[Here's the file](https://d.pr/f/snDXFa). In this file i have two artboards; one with a containing group, one with just a button on an artboard. The [same bug seems to occur](https://d.pr/i/kfi7s6).
